WordPress/Classes/ViewRelated/Me/AccountSettingsViewController.swift

```diff
@@ -60,7 +60,7 @@ private struct AccountSettingsController: SettingsController {
         let email = EditableTextRow(
             title: NSLocalizedString("Email", comment: "Account Settings Email label"),
             value: settings?.email ?? "",
-            action: presenter.present(insideNavigationController(editEmailAddress(service)))
+            action: presenter.present(insideNavigationController(insidePromptController(editEmailAddress(service))))
```

Comment:
I've already mentioned that I think the prompt should include its own navigation controller, but I also think `ImmuTablePresenter` should provide `prompt` in an extension to reduce verbosity, but also for type safety: `ImmuTablePresenter.prompt(_:)` would also have to be generic and take a `<T: UIViewController where T: PresentedViewController>`

```swift
presenter.prompt(editEmailAddress(service))
```